tutorials/nservicebus-101/lesson-5/tutorial.md

```diff
@@ -167,18 +167,26 @@ Notice how much faster the message proceeds through delayed retries, because ins
 
 If you used the [Particular Platform Installer](/platform/installer/) to install MSMQ, you should already have the [ServiceControl](/servicecontrol/) and [ServicePulse](/servicepulse/) tools installed. You may want to re-run the Platform Installer and ensure that the checkboxes for ServiceControl and ServicePulse are already checked, and install them if necessary.
 
+NOTE: ServiceControl is installed as a Windows service named **Particular ServiceControl** and has no user interface. ServicePulse is installed as a Windows service named **Particular ServicePulse** and has a web-based UI, which can be accessed at [http://localhost:9090](http://localhost:9090) when default settings are used. You can check to see if both are running from the Windows Services control panel.
+
 Once these tools are installed and running, you can attempt to replay a message:
 
  1. Fix the **Sales** endpoint by removing the `throw` statement.
  1. Run the solution.
- 1. Open ServicePulse and navigate to the **Failed Messages** page. Note how similar messages are grouped together for easier handling.
+ 1. Open ServicePulse at [http://localhost:9090](http://localhost:9090) and navigate to the **Failed Messages** page. Note how similar messages are grouped together for easier handling.
```

Comment:
make http://localhost:9090 code and not a link so it wont show up as a broken link when we link check

also link to SP port/hosting doco https://docs.particular.net/servicepulse/host-config